equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/util/DateUtil.java

```diff
@@ -43,4 +47,18 @@ public static Date convertStringToSimpleDate(String dateToConvert) throws ParseE
 
         return new SimpleDateFormat(DATE_FORMAT_PATTERN, Locale.UK).parse(dateToConvert);
     }
-}
+
+    public static String calculateTimeDifference(String startDateTime, String endDateTime, String format) {
```

Comment:
`protected` instead of public - visible for the package and all subclasses of the declaring class

Reply:
this method should be accessible from historicaldata.mapper package, so changing it to protected will make it invisible.